model_info/cifar100/corruptions/unaggregated_results.csv

```diff
@@ -10,3 +10,4 @@ Diffenderfer2021Winning_Binary,0.775,0.7663,0.7552,0.7393,0.6926,0.7676,0.731,0.
 Diffenderfer2021Winning_Binary_CARD_Deck,0.7855,0.7767,0.7644,0.7451,0.6893,0.7759,0.7198,0.6681,0.5857,0.3261,0.7827,0.7705,0.7541,0.7287,0.6759999999999999,0.7357,0.7414,0.7289,0.6844,0.6317,0.7836,0.7472,0.7023,0.6265,0.4314,0.7548,0.7126,0.6598,0.6597,0.6001,0.7553,0.7304,0.7058,0.691,0.6814,0.6048,0.6125,0.6329,0.4737,0.5094,0.7633,0.7335,0.7047,0.6216,0.5325,0.7478,0.7191,0.7049,0.6915,0.6774,0.7572,0.7303,0.6945,0.6936,0.655,0.7728,0.7573,0.7429,0.7002,0.5959,0.7677,0.7533,0.7239,0.716,0.6867,0.7592,0.7016,0.6799,0.6409,0.6184,0.753,0.7471,0.733,0.7139,0.6824
 Addepalli2021Towards_PARN18,0.6211,0.6115,0.5906,0.5595,0.4787,0.5629,0.3907,0.2686,0.14300000000000002,0.0373,0.6104,0.5829,0.5523,0.5304,0.4789,0.5529999999999999,0.5536,0.5365,0.5294,0.5209,0.5897,0.4889,0.3733,0.2549,0.1087,0.5859,0.5368,0.4695,0.4705,0.4164,0.6182,0.6038,0.5716,0.5417,0.5124,0.5705,0.5737,0.5565,0.5156,0.5163,0.5998,0.5624,0.5147,0.4023,0.294,0.6041,0.5953,0.5895,0.5868,0.5796,0.5761,0.5382,0.4999,0.4975,0.4624,0.6132,0.603,0.5997,0.5869,0.5629,0.6205,0.6168,0.5877,0.5649,0.5113,0.6122,0.5904,0.564,0.5242,0.4817,0.564,0.557,0.541,0.5303,0.5081
 Addepalli2021Towards_WRN34,0.6612,0.6483,0.629,0.601,0.5244,0.601,0.413,0.2831,0.1501,0.0367,0.6479,0.6225,0.5948,0.5679,0.5046,0.5824,0.5841,0.5723,0.5581,0.5497,0.6252,0.521,0.3998,0.2813,0.1164,0.6244,0.5759,0.508,0.507,0.4457,0.6502,0.6324,0.5878,0.5586,0.5251,0.5992,0.5995,0.5866,0.5378,0.5375,0.6286,0.592,0.551,0.4678,0.3738,0.6358,0.6241,0.6194,0.6153,0.6076,0.6104,0.5701,0.5238,0.5243,0.4813,0.6477,0.6413,0.6321,0.6204,0.5883,0.6534,0.65,0.6141,0.5872,0.5317,0.6522,0.6274,0.6021,0.5634,0.5248,0.5955,0.5896,0.5718,0.5582,0.5327
+Modas2021PRIME_RN18_100,0.7739,0.7698,0.7601,0.7442,0.7095,0.7709,0.7508,0.7316,0.7032,0.604,0.7733,0.7593,0.7371,0.707,0.639,0.7184,0.7261,0.7147,0.6767,0.6272,0.7717,0.7484,0.7074,0.6518,0.4926,0.7423,0.7127,0.6769,0.6801,0.6395,0.7315,0.7068,0.6725,0.6565,0.6407,0.623,0.6319,0.6361,0.5417,0.5471,0.7361,0.68,0.6207,0.5216,0.4336,0.7184,0.677,0.6553,0.6399,0.6178,0.7458,0.7153,0.6769,0.6785,0.6433,0.7563,0.7286,0.7057,0.6267,0.5212,0.7488,0.7284,0.7001,0.6911,0.6648,0.7527,0.7045,0.6824,0.6457,0.6284,0.7428,0.7341,0.718,0.697,0.6658
```

Comment:
I think that here the same comment that Maksym made below about the `_100` postfix applies

Reply:
Oh, I didn't see the lastest comment by @max-andr, who suggested exactly the same thing